templates/terraform/sweeper_file.go.erb

```diff
@@ -0,0 +1,115 @@
+<%= lines(autogen_notice :go) -%>
+
+package google
+
+import (
+  "fmt"
+  "testing"
+  "strings"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+<%
+sweeper_name = product_ns + object.name
+wrap_path = object&.nested_query&.keys&.first || object.collection_url_key
+-%>
+
+func init() {
+	resource.AddTestSweepers("<%= sweeper_name -%>", &resource.Sweeper{
+		Name: "<%= sweeper_name -%>",
+		F:    testSweep<%= sweeper_name -%>,
+	})
+}
+
+func testSweep<%= sweeper_name -%>(region string) error {
```

Comment:
Our CI build calls it for us-central1 only. I figured we could attempt to grab global/aggregated zonal resources when we can, otherwise just stick to what was passed in from the command. I suppose we modify the CI build?

Reply:
SGTM- can you comment that at the time of writing, we expect CI to only pass in `us-central1`? Checking all the regions for all the resources could take awhile, so keeping this as-is is fine. I think we use a few more fairly often, but we should modify CI if we want to sweep them too